roles/confluent.variables/defaults/main.yml

```diff
@@ -529,6 +529,10 @@ kafka_connect_key_path: "/var/ssl/private/kafka_connect.key"
 kafka_connect_export_certs: "{{kafka_connect_ssl_mutual_auth_enabled}}"
 kafka_connect_keytab_path: /etc/security/keytabs/kafka_connect.keytab
 kafka_connect_kafka_listener_name: internal
+
+### Additional extension classes to be added to 
```

Comment:
This comment should say what format this variable needs to be in, It appears it should be a comma separated string
Do you like comma separated string over list?

also run python doc.py to get the variable into variables.md

Reply:
Good point. I'd say in this case comma-separated over list is simpler.